x-pack/test/functional_execution_context/tests/browser.ts

```diff
@@ -316,7 +316,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
                 type: 'search',
                 name: 'discover',
                 id: '571aaf70-4c88-11e8-b3d7-01146121b73d',
-                description: '[Flights] Flight Log',
+                description: '',
```

Comment:
The `description` comes from the embeddable title. Previously, the title value was set from runtime state which loaded the saved object title as this is a by-reference panel. Now, titles are set only by serialized state, so the title is undefined.